### PR TITLE
Use embedded configuration by default

### PR DESCRIPTION
**Pull request by sbc100**
I can't see any reason why the embedded mode isn't always preferable.
The emsdk is designed to be used as-is, and per-user customizations
don't make sense.

I'm hoping if this sicks we can just remove this options completely.

This is part of a wider plan to remove the use of the user's HOME
directory completely:
  https://github.com/emscripten-core/emscripten/issues/9543

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -54,7 +56,9 @@ jobs:
           name: Install python 3
           command: brew install python3
       - run: ./test.sh
-      - run: ./test.py
+      - run: |
+         - source emsdk_env.sh
```
Comment (kripken) [inline .circleci/config.yml:60]: is this change required because of this PR?

Comment (sbc100) [reply .circleci/config.yml:60]: I should be able to revert that, now that https://github.com/emscripten-core/emscripten/pull/10935 has landed.

Technically I think the tests are wrong here because they should run the context in whic `emsdk_env` has been run.   i.e. the tests were not following the instructions here.

**Comment by sbc100** [comment]
I'm going to write the mailing list about this change now.. and hopefully is now good to go?
